### PR TITLE
data: sejosephmartin

### PR DESCRIPTION
**Pull request by sejosephmartin**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6351"></a>

### CODE REVIEW

File: sejosephmartin.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Joseph Martin",
+  "displayStatsPublic": true,
```
[the remark this answers is not in the document]
Comment (imbeshat) [reply sejosephmartin.json:3]: not required

File: sejosephmartin.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "type": "personal",
+  "bio": "Aspiring Front end developer 🚀 | Self-Taught Developer ✨ | Undergraduate"
+  "avatar": "https://github.com/sejosephmartin.png"
```
Comment (imbeshat) [inline sejosephmartin.json:6]:
```suggestion
```

Comment (imbeshat) [reply sejosephmartin.json:6]: not required

File: sejosephmartin.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Joseph Martin",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Aspiring Front end developer 🚀 | Self-Taught Developer ✨ | Undergraduate"
```
Comment (imbeshat) [inline sejosephmartin.json:5]:
```suggestion
  "bio": "Aspiring Front end developer 🚀 | Self-Taught Developer ✨ | Undergraduate",
```

**Review by imbeshat — CHANGES_REQUESTED** [review]
Also, move your file to ```data``` folder.

**Comment by sejosephmartin** [comment]
> Also, move your file to `data` folder

can I move the file like this  **data/sejosephmartin.json.**. I change already some of the unnecessary script. should i create another branch when i commit this?

**Review by khushalarora — APPROVED** [review]
Looks good, thanks :tada:

**Review by harshsingh32 — COMMENTED** [review]
lgtm

**Review by kumarsonsoff3 — APPROVED** [review]
Looks awesome. Great collaboration everyone. 👍

**Pull request merged**